src/averroes/soot/CodeGenerator.java

```diff
@@ -552,6 +591,11 @@ private void callApplicationMethodsReflectively() {
 		}
 
 	}
+	private boolean isLifeCycle(SootMethod method) {
+		// TODO: Refactor (e.g. store the method signatures inside this class)?
+		AndroidEntryPointUtils m = new AndroidEntryPointUtils();
```

Comment:
What's AndroidEntryPointUtils?

Reply:
Class containing common utility methods for dealing with Android entry points. (part of soot)

Reply:
I see.